tests/test_agent.py

```
@@ -0,0 +1,263 @@
+        model = MockModel()
+        agent = create_agent(
+            name="schema-agent",
+            description="Agent for testing tool parameters",
+            persona="You are a test assistant",
+            model=model,
+            tools=[tool],
+        )
+
+        assert len(agent.tools) == 1
+        assert "sample" in agent.tools
+        assert agent.tools["sample"].input_parameters["x"] == int
```

Ruff (0.14.10)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)